kit/src/components/context_menu/context.js

```diff
@@ -1,37 +1,53 @@
+let click = (toggle) => function (ev) {
+  ev.stopPropagation()
+  ev.preventDefault()
+  // Hide any open context menus
+  const menus = document.getElementsByClassName("context-menu")
+  // Select the current one
+  const context_menu = document.getElementById("UUID-context-menu")
+  let hidden = context_menu.classList.contains("hidden");
+  for (var i = 0; i < menus.length; i++) {
+    menus.item(i).classList.add("hidden")
+  }
+  if (toggle && !hidden) {
+    context_menu.classList.add("hidden")
+    return
```

Comment:
this should probably have a boolean return type.

Reply:
actually the end return is not needed instead